src/request-api.js

```diff
@@ -33,24 +34,18 @@ function onRes (buffer, cb) {
 
     if (chunkedObjects) {
       const parsed = []
-      res.on('data', chunk => parsed.push(JSON.parse(chunk)))
+      res.on('data', chunk => {
+        try {
+          parsed.push(JSON.parse(chunk))
+        } catch (err) {
+          // Browser quirks emit more than needed sometimes
```

Comment:
if you have 3 chunks, you sometimes get: 
* chunk1
* chunk2
* chunk3
* chunk1+chunk2 => not parsable

do not ask me why, browsers are a weird bunch..